opencodeblocks/graphics/blocks/block.py

```diff
@@ -133,8 +171,16 @@ def setTitleGraphics(self, color: str, font: str,
             padding: title padding.
 
         """
-        self.title_widget.setMargin(int(padding))
-        self.title_widget.setStyleSheet(f"QLabel {{ color : {color} }}")
+        # self.title_widget.setMargin(int(padding))
+        self.title_widget.setStyleSheet(
+            f"""
+            QLineEdit {{
+                color : {color};
+                background-color: #E3212121;
```

Comment:
Why the background color ?
I think having a transparent background is better if we modify block color someday

Reply:
It's so that the title does not stand out. It's so that the appearance of the title does not change with this version.

Reply:
![image](https://user-images.githubusercontent.com/60117466/143938425-a6d46220-48a5-4a78-afc2-59f11b82ac6f.png)
Well it does stand out ! :P

Reply:
```suggestion
                background-color: transparent;
```